### PR TITLE
HS64 V2-re

### PR DESCRIPTION
**Pull request by SnowLire**

### CODE REVIEW

**Review by InfinityGhost — CHANGES_REQUESTED** [review]
File extension for the configuration should be `.json`, OpenTabletDriver ignores anything without it.

**Comment by SnowLire** [comment]
Fixed

File: OpenTabletDriver/Configurations/Huion/HS64 V2.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "Name": "Huion HS64 V2",
```
Comment (InfinityGhost) [inline OpenTabletDriver/Configurations/Huion/HS64 V2.json:2]: Is this really a V2 tablet or is it another identifier for the tablet?

Comment (SnowLire) [reply OpenTabletDriver/Configurations/Huion/HS64 V2.json:2]: Afaik it's a different device string so I went with the safe route and make a different config.

Comment (InfinityGhost) [reply OpenTabletDriver/Configurations/Huion/HS64 V2.json:2]: Makes more sense to just add another identifier IMO, as long as the specifications are the same there's no reason for it to change.

Comment (SnowLire) [reply OpenTabletDriver/Configurations/Huion/HS64 V2.json]: Done.

**Comment by InfinityGhost** [comment]
Migrate to the new configuration directory.

**Pull request merged**